mgmt/imgmgr/src/imgmgr.c

```diff
@@ -340,6 +356,45 @@ imgr_erase(struct mgmt_cbuf *cb)
     return 0;
 }
 
+static int
+imgr_erase_state(struct mgmt_cbuf *cb)
+{
+    const struct flash_area *fa;
+    int area_id;
+    int rc;
+    CborError g_err = CborNoError;
+
+    area_id = imgmgr_find_best_area_id();
+    if (area_id >= 0) {
+        rc = flash_area_open(area_id, &fa);
+        if (rc) {
+            return MGMT_ERR_EINVAL;
+        }
+
+        rc = flash_area_erase(fa, 0, sizeof(struct image_header));
```

Comment:
Ignoring this rc? Looks like there's a similar issue in imgr_erase()...

Reply:
added also commit that fixes this in `imgr_erase()`